src/styles/styles.scss

```diff
@@ -1,7 +1,11 @@
+@font-face {
+  font-family: Helvetica;
+  src: url(../assets/fonts/Helvetica_Neue_LT_Std-W55_Normal.otf);
```

Comment:
I love Helvetica. Nice that you were able to grab it!

Reply:
Yeah, it's part of Savile!!